webextensions/api/proxy.json

```diff
@@ -530,7 +531,10 @@
                 "version_added": false
               },
               "firefox": {
-                "notes": "In version 59, this property was listed as <code>proxyConfig</code> in the <code>browserSettings</code> namespace, but it had a bug that made it mostly unusable.",
+                "notes": [
+                  "In version 59, this property was listed as <code>proxyConfig</code> in the <code>browserSettings</code> namespace, but it had a bug that made it mostly unusable.",
+                  "From version 88 the <code>ftp</code> setting has no effect, because FTP is not longer supported (see <a href='https://bugzil.la/1626365'>bug 1626365</a>)."
```

Comment:
The old note is a bit of a nuisance, really (it covers a version not listed in the range). How about we get rid of it and sub in your note? We can even put it after the "version_added" field, which is conventional, though not strictly required.

(There's also a couple of tiny copy edits in the note itself)

```
"version_added": "60",
"notes": "From version 88, the <code>ftp</code> setting has no effect because FTP is no longer supported (see <a href='https://bugzil.la/1626365'>bug 1626365</a>)."

Reply:
Makes sense!